templates/step_function/consignment_export_definition.json.tpl

```diff
@@ -129,4 +216,4 @@
       "Type": "Fail"
     }
   }
-}
+}
```

Comment:
Missing line break at end of file

Reply:
I had Intellij configured to add these in and it seems to have stopped :frowning_face: